### ISSUE TITLE
DB Error: Field 'dt' doesn't have a default value REPLACE INTO stats_lastseen SET `user` = 'user'

### ISSUE THREAD

**Issue by nickgreen43** [splitbrain/dokuwiki-plugin-statistics#64]
This error keeps showing when logging in for every user in our system. It's caused by the Statistics plugin (https://www.dokuwiki.org/plugin:statistics). 

I tried following this https://github.com/splitbrain/dokuwiki-plugin-statistics/issues/50 but I was not able to find the my.cnf config file.

Re-installing didn't help either. Only disabling the plugin makes the error go away.

**Comment by nickgreen43** [comment]
I think I found a resolution to the error... since the "stats_lastseen" table (from https://github.com/splitbrain/dokuwiki-plugin-statistics/blob/master/db.sql) has  NOT NULL but no default placed, it was erroring out. I added the below statement to alter the table to be more like the 'stats_access' table. Can't say if there are repercussions yet or not but so far so good at least the error is not showing up on the dokuwiki for users anymore. 

` ALTER TABLE `stats_lastseen` CHANGE COLUMN `dt` `dt` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
`
